evap/evaluation/models.py

```diff
@@ -958,6 +959,14 @@ def annotate_with_participant_and_voter_counts(cls, evaluation_query):
             num_voters=Subquery(voter_count_subquery),
         )
 
+    @classmethod
+    def get_evaluation_url_tuples_to_review_urgent(cls) -> List[Tuple['Evaluation', str]]:
+        return [
+            (evaluation, f"{PAGE_URL}/staff/semester/{evaluation.course.semester.id}/evaluation/{evaluation.id}/textanswers")
```

Comment:
I think we should use django's [`reverse`](https://docs.djangoproject.com/en/dev/ref/urlresolvers/#reverse) method here